packages/radix-ui-themes/src/components/avatar.css

```diff
@@ -25,11 +25,6 @@
   justify-content: center;
   line-height: 1;
   border-radius: inherit;
-
-  & svg {
-    width: var(--avatar-font-size-one-letter);
-    height: var(--avatar-font-size-one-letter);
-  }
```

Comment:
We shouldn't set svg size imo, it can be quite different depending on the design you are after